examples/code/shader_toy/shader_toy.cpp

```diff
@@ -287,7 +287,10 @@ namespace
 
         init_renderer();
 
-        put::dev_ui::init();
+        ImGuiStyle &style = ImGui::GetStyle();
```

Comment:
put::dev_ui::init(); can handle this with the default argument, I expect to see no changes to this file in this pull request.